var/spack/repos/builtin/packages/py-pauvr/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPauvr(PythonPackage):
+    """pauvr: a plotting package designed for nanopore and PacBio long reads"""
+
+    homepage = "https://github.com/conchoecia/pauvre"
+    git      = "https://github.com/conchoecia/pauvre.git"
+
+    version('develop', branch='develop')
+    version('master',  branch='master')
+    version('0.1.86',  tag='0.1.86')
```

Comment:
I think it would be preferred to have the release pull from https://github.com/conchoecia/pauvre/archive/0.1.86.tar.gz instead of the git tag, and also for it to be listed first, since it is the 'preferred' version.

Reply:
did